sci-libs/vtk/metadata.xml

```diff
@@ -1,17 +1,21 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-  <maintainer type="project">
+  <maintainer type="project" proxied="proxy">
```

Comment:
But you didn't add yourself?

Reply:
No, I'm not the maintainer of the package. I can add myself as a secondary maintainer if you insist. Having the package for a long time in my repo and fixed issues with it in the past, but only in my overlay, so I have some insight into the building process by now. But I don't want to be primary maintainer, as the package is quite complex and has long build times on my machine, especially with the new cuda support enabled.

Reply:
No problem.

Reply:
I watch about the package and will take some care of it. But currently, I don't want to be listed as the only non-project maintainer for the package. It needs too much resources.

Reply:
I now added myself as a maintainer. The CI bot was unhappy about having no proxied maintainer and left with a broken package state.